base/applications/games/solitaire/lang/cs-CZ.rc

```diff
@@ -13,13 +17,13 @@ STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
 CAPTION "Možnosti"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Karty", -1, 7, 7, 90, 40
+    GROUPBOX "Sejm&out", -1, 7, 7, 90, 40
```

Comment:
```suggestion
    GROUPBOX "Sejmout", -1, 7, 7, 90, 40
```
Note that the original translation more closely followed the generic "cards" translation, instead of focusing on the action of drawing them.

Reply:
I'm going to commit your PR like this. If later you think that the previous translation ("Karty") was better, you can make the change in a future translation PR with other new translations.